src/main/java/Application.java

```diff
@@ -0,0 +1,13 @@
+import Controller.LadderGame;
+ import view.InputView;
+import view.OutputView;
+import view.printer.Printer;
+import view.reader.Reader;
+
+public class Application {
+    public static void main(String[] args) {
+        Printer printer = new Printer();
+        LadderGame ladderGame = new LadderGame(InputView.of(new Reader(), printer), OutputView.from(printer));
```

Comment:
```suggestion
        Printer printer = new Printer();
        Reader reader = new Reader();
        LadderGame ladderGame = new LadderGame(InputView.of(reader, printer), OutputView.from(printer));
```
reader도 변수 추출을 해봅시다.

Reply:
`new Reader()`가 한번만 쓰이기 때문에 인라인으로 해주었는데 혹시 가독성이 많이 떨어진다고 느껴질까요?

Reply:
아하 printer는 두 번 사용되는군요! 저라면 변수 추출을 해줄 것 같지만 취향의 영역으로 보이네요. 지금도 괜찮을 것 같습니다!